packages/prisma/migrations/20230303195432_add_feature_flag_default_values/migration.sql

```diff
@@ -0,0 +1,29 @@
+-- Insert initial feature flags with their default values
```

Comment:
Since we don't have UI to add new keys and we need to code them anyways. We should add them via migrations.